colabs/wandb_registry/zoo_wandb.ipynb

```diff
@@ -0,0 +1,984 @@
+{
```

Comment:
<ul><li>There are two ways to get the target path, "interatively" -&gt; "interactively"</li><li>Not sure it's even necessary to add the commented out interactive method of getting the target path here, but will leave that up to the author. </li><li>The target path of a "collectin" consists of three parts -&gt; "collection"</li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/wandb/examples/pull/560/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>